sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SpanLimits.java

```diff
@@ -44,66 +44,66 @@ public static SpanLimitsBuilder builder() {
   }
 
   static SpanLimits create(
-      int maxNumAttributes,
-      int maxNumEvents,
-      int maxNumLinks,
-      int maxNumAttributesPerEvent,
-      int maxNumAttributesPerLink,
-      int maxAttributeLength) {
+      int spanAttributeLimit,
+      int spanEventLimit,
+      int spanLinkLimit,
+      int eventAttributeLimit,
+      int eventLinkLimit,
+      int attributeValueLengthLimit) {
     return new AutoValue_SpanLimits(
-        maxNumAttributes,
-        maxNumEvents,
-        maxNumLinks,
-        maxNumAttributesPerEvent,
-        maxNumAttributesPerLink,
-        maxAttributeLength);
+        spanAttributeLimit,
+        spanEventLimit,
+        spanLinkLimit,
+        eventAttributeLimit,
+        eventLinkLimit,
+        attributeValueLengthLimit);
   }
 
   /**
    * Returns the global default max number of attributes per {@link Span}.
    *
    * @return the global default max number of attributes per {@link Span}.
    */
-  public abstract int getMaxNumberOfAttributes();
+  public abstract int getSpanAttributeLimit();
 
   /**
    * Returns the global default max number of events per {@link Span}.
    *
    * @return the global default max number of events per {@code Span}.
    */
-  public abstract int getMaxNumberOfEvents();
+  public abstract int getSpanEventLimit();
```

Comment:
The word "Span" in these methods is probably redundant, since we're in the SpanLimits class.